index.js

```diff
@@ -0,0 +1,130 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const util = require('util');
+
+const core = require('@actions/core');
+const toolCache = require('@actions/tool-cache');
+
+/** Get the expected file extension for the cr binary. */
+function getExecutableExtension() {
+  if (os.type().match(/^Win/)) {
+    return '.exe';
+  } else {
+    return '';
+  }
+}
```

Comment:
aren't we explicitly running the actions on ubuntu-latest? Do we still want this check anyway as best practice?

Reply:
I carried this pattern over from setup-helm. We could have said the action only works on ubuntu, but it wasn't that much extra work to support the different OS-es.